README.md

```diff
@@ -97,7 +97,3 @@ details.
 You should have received a copy of the GNU Affero General Public License along
 with this program. If not, see https://www.gnu.org/licenses/
 ```
-
-[CONTRIBUTING]: CONTRIBUTING.md
```

Comment:
With caveats of a) the content changes are more important than this tiny style issue, b) I only sort of care about this, c) I think consistency is more important and I see you've changed it everywhere and that's nice ... let me submit a tiny vote in favor of preserving the "footnote style" on all the links here.

IMO it makes reading/scanning the document (as a markdown document in an editor) much much easier, and it makes the diffs to change links much more targeted to a single line, while keeping the rendered output on GH or elsewhere the same.

Reply:
Thanks for the comment. There were only a couple of the footnote style references in this doc, so I went with making it consistent in the direction of inline links. We could consider adding style guidance for Markdown somewhere. I recognise that the inline links can be a distraction when reading though.